src/reconcile/rec_write.c

```diff
@@ -1150,13 +1150,16 @@ __rec_split_grow(WT_SESSION_IMPL *session, WT_RECONCILE *r, size_t add_len)
     return (0);
 }
 
+/* The minimum number of entries before we'll split a row-store internal page. */
+#define WT_PAGE_INTL_MINIMUM_ENTRIES 20
```

Comment:
How is this calculated?

Reply:
That's what it was before this change, there was a hard-coded constant in the reconciliation code that required at least 20 items on an internal page: https://github.com/wiredtiger/wiredtiger/blob/04f81c08ac93fb1f006b4f4b47ce1d6ebb86886d/src/reconcile/rec_row.c#L494